### ISSUE TITLE
Columns to be removed from siteselect

### ISSUE THREAD

**Issue by ljarvi** [Urban-Meteorology-Reading/SUEWS-Docs#20]
**Describe the Issue**
Columns 54-57 should be removed from SiteSelect.txt and be added to SUEWS_AnthropogenicHeat.txt

**Location**
SiteSelect.txt and SUEWS_AnthropogenicHeat.txt

**Screenshots**

**Additional context**

**Comment by biglimp** [comment]
There kind of changes should be done with care as it will affect users and developers in multiple ways...

**Comment by sunt05** [comment]
This is already effective in the v2018a files: only the descriptions in this manual are incorrect. Now the manual and actual files should be consistent. We'll document this in the `Version History`.

**Comment by ljarvi** [comment]
I think the main change related to the anthopogenic profiles is that four of them were moved from SiteSelect to AnthopogenicHeat. I see that in AnthopogenicHeat.txt description you have TODO reading on those columns related to CO2 and QF parts. I added decriptions here: https://suews-docs.readthedocs.io/en/latest/input_files/SUEWS_SiteInfo/Input_Options.html#cmdoption-arg-minqfmetab for each of the components. Mainly reading Not used in V2018a.

Leena
